src/components/layout/Navbar/style.module.scss

```diff
@@ -77,7 +89,86 @@
     background: vars.$wainbow;
     height: 0.25rem;
     position: absolute;
-    top: 4rem;
+    top: 4rem; // Position the wainbow at the bottom
+    transform: translateY(0); // Move wainbow to bottom of navbar by default
+    transition: 0.3s ease-in-out; // Apply transition to transform property
     width: 100vw;
   }
+
+  .mobileNav {
+    align-items: right;
```

Comment:
Should be `flex-end` instead of `right`

valid properties for this are `flex-start`, `center`, and `flex-end`, 
![Screenshot 2023-05-08 at 10 05 24 AM](https://user-images.githubusercontent.com/33165426/236885995-934b7702-42fc-49a4-9f94-af40943b1119.jpg)